### ISSUE TITLE
Math anything after "/"

### ISSUE THREAD

**Issue by evgeny-matveev** [sporto/hop#32]
Hello, I'm looking for a way to match everything after "/" in the address bar, e.g. if I go to http://my-site.com/#/home/evgeny/Downloads/images/pizza I expect that "home/evgeny/Downloads/images/pizza" is the _str_ in:
```
matchFolder =
  match2 FolderRoute "/" str
```

And then I could make a request to my back-end: http://my-api.com/api/home/evgeny/Downloads/images/pizza Does it make any sense?

**Comment by sporto** [comment]
Tried, this doesn't work, anything with slashes in the middle won't match. I will need to investigate.

**Comment by sporto** [comment]
I have published 5.1. which now includes a `regex` matcher. This allows for this case. You use it like:

```
catchAllRoute =
    match2 Other "/" (regex ".+")
 ```
